visallo/visallo
# Make within the default geo predicate + fill values after first click(4.0)

Testing Instructions: 
- Within should be the option when you first add a geolocation or any geolocation property filter
- The property filter should get values when you first click on the map popover and see a circle.

CHANGELOG
Changed: When specifying a geolocation search filter with a map popover, the filter is now added immediately after the first click, rather than after the first resize.
